app/components/UserAttendance/AttendanceModal.tsx

```diff
@@ -1,6 +1,6 @@
 import cx from 'classnames';
-import { flatMap } from 'lodash';
-import { Component } from 'react';
+import { flatMap } from 'lodash-es';
```

Comment:
We have some webpack config to convert all lodash imports to lodash-es, so you can just to `lodash`.

Reply:
Ye I know, I just wanted to be precise for some reason 🤷🏼